### PR TITLE
Add GitHub templates for Issue and Pull Request

### PR DESCRIPTION
**Pull request by acjh**
**What is the purpose of this pull request? (put an "X" next to item)**

- [ ] Documentation update
- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement to an existing feature
- [x] Other, please explain: Improvement of Issues and Pull Requests 

**What is the rationale for this request?**

> Currently, Markbind does not have any structure that contributors are advised to follow. This makes it harder to determine if it is indeed a bug or intended behavior or caused by a difference in expectation of the user and the developer.
— [@rachx](https://github.com/nus-cs3281/2018/blob/master/students/processObservations.md#rachael-sim-hwee-ling)

> If we have a good format, we will have less confusion/discussion about the feature details.
— [@nicholaschuayunzhi](https://github.com/nus-cs3281/2018/blob/master/students/processObservations.md#chua-yun-zhi-nicholas)

**What changes did you make? (Give an overview)**

- bf06b7b Add GitHub issue template
- f550610 Add GitHub pull request template

**Provide some example code that this change will affect:**

\-

**Is there anything you'd like reviewers to focus on?**

Phrasing, grammar and verbosity. Feel free to nitpick as this is what all our contributors will see.

**Testing instructions:**

The templates are rendered in the comments below: [Issue template](https://github.com/MarkBind/markbind/pull/208#issuecomment-385240197) and [Pull Request template](https://github.com/MarkBind/markbind/pull/208#issuecomment-385240203).
You may attempt to edit/preview the comments to see the `<!-- hints in HTML comments -->`.

### CODE REVIEW

**Comment by acjh** [comment]
# Issue template

**Tell us about your environment**

* **MarkBind Version:**

**What did you do? Please include the actual source code causing the issue.**


```html

```

**What did you expect to happen?**


**What actually happened? Please include the actual, raw output.**

**Comment by acjh** [comment]
**What is the purpose of this pull request? (put an "X" next to item)**

- [ ] Documentation update
- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement to an existing feature
- [ ] Other, please explain:






**What is the rationale for this request?**


**What changes did you make? (Give an overview)**


**Provide some example code that this change will affect:**


```js

```

**Is there anything you'd like reviewers to focus on?**


**Testing instructions:**

**Comment by acjh** [comment]
Hmm, the Markdown checklist results in a misleading "1 of 5" progress:

screenshot 2018-04-30 18 13 12

Could be why ESLint uses plain square brackets, e.g.:

[ ] Documentation update
[ ] Bug fix
[ ] New feature
[ ] Enhancement to an existing feature
[x] Other, please explain: Improvement of Issues and Pull Requests

**Comment by acjh** [comment]
@damithc I need [Admin permissions](https://help.github.com/articles/repository-permission-levels-for-an-organization/) to set up the (new) [prompt to choose from multiple issue types](https://blog.github.com/2018-05-02-issue-template-improvements/).

**Comment by acjh** [comment]
Shall go with:

> **What is the purpose of this pull request? (put "X" next to an item, remove the rest)**
>
> • [ ] Documentation update
> • [ ] Bug fix
> • [ ] New feature
> • [ ] Enhancement to an existing feature
> • [x] Other, please explain: Improvement of Issues and Pull Requests

**Comment by damithc** [comment]
> @damithc I need Admin permissions to set up the (new) prompt to choose from multiple issue types.

Gave you admin permissions

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,39 @@
+**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**
+
+• [ ] Documentation update
+• [ ] Bug fix
+• [ ] New feature
+• [ ] Enhancement to an existing feature
+• [ ] Other, please explain:
```
Comment (damithc) [inline .github/PULL_REQUEST_TEMPLATE]: `• [ ] Documentation update` -> `- [ ] Documentation update` ?

Comment (acjh) [reply .github/PULL_REQUEST_TEMPLATE]: I changed from `- [ ]` to `• [ ]` in 6bfec64 for the reason in https://github.com/MarkBind/markbind/pull/208#issuecomment-385359556.

Comment (damithc) [reply .github/PULL_REQUEST_TEMPLATE]: Got it.

**Pull request merged**